elements/lisk-validator/src/validation.ts

```diff
@@ -242,3 +242,18 @@ export const validateFee = (data: string): boolean =>
 	isNumberString(data) &&
 	isGreaterThanZero(new BigNum(data)) &&
 	!isGreaterThanMaxTransactionAmount(new BigNum(data));
+
+export const isCsv = (data: string): boolean => {
+	const maxItemCount = 1000;
```

Comment:
I don't thinks its a good idea to mention max item count for a generic formatter like `isCSV`. This should and only check if data is a csv or not. If you want to limit CSV items for particular use case then create specific format for it. e.g. `isValidListOfKeys`. Or this is right use case to introduce a `keyword` e.g. `CSVMaxItem`. 

One can also specify a regular expression as `pattern` directly on the JSON-schema to limit the items in CSV string.

Reply:
This honours the original implementation in framework. I think this is outside the scope of this PR for which only methods that were missing were added to this library. Please open an issue with suggestions for lisk-validator

Reply:
The above code is introduced in the current PR, so its the right scope to do it right. This validator `isCsv` is generic one, it must not have such limits in it as explained above.

Reply:
Code was originally in framework I just moved it but made the changes. Check is now done in validation.ts and also in the blocksCommon function.